COLLABORATOR_GUIDE.md

```diff
@@ -411,8 +411,11 @@ recommended but not required.
 
 ### Deprecations
 
-**Deprecation** refers to the identification of Public APIs that should no
-longer be used.
+[_Deprecation_] is "the discouragement of use of some…feature…or practice,
```

Comment:
What's the difference between `[_Deprecation_]` and `[_Deprecation_][]`? I have seen both versions, but mostly the latter.

Reply:
I'd not noticed that the former worked. I meant to do the latter but made a mistake. I'll fix that.